core/templates/domain/skill/MisconceptionObjectFactorySpec.ts

```diff
@@ -20,35 +20,84 @@ import { MisconceptionObjectFactory } from
   'domain/skill/MisconceptionObjectFactory';
 
 describe('Misconception object factory', () => {
-  describe('MisconceptionObjectFacfory', () => {
-    let misconceptionObjectFactory: MisconceptionObjectFactory;
-    let misconceptionDict: any;
-
-    beforeEach(() => {
-      misconceptionObjectFactory = new MisconceptionObjectFactory();
-      misconceptionDict = {
-        id: '1',
-        name: 'test name',
-        notes: 'test notes',
-        feedback: 'test feedback',
-        must_be_addressed: true
-      };
-    });
-
-    it('should create a new misconception', () => {
-      var misconception =
-        misconceptionObjectFactory.createFromBackendDict(misconceptionDict);
-      expect(misconception.getId()).toEqual('1');
-      expect(misconception.getName()).toEqual('test name');
-      expect(misconception.getNotes()).toEqual('test notes');
-      expect(misconception.getFeedback()).toEqual('test feedback');
-      expect(misconception.isMandatory()).toEqual(true);
-    });
-
-    it('should convert to a backend dictionary', () => {
-      var misconception =
-        misconceptionObjectFactory.createFromBackendDict(misconceptionDict);
-      expect(misconception.toBackendDict()).toEqual(misconceptionDict);
-    });
+  let misconceptionObjectFactory: MisconceptionObjectFactory;
+  let misconceptionDict: any;
+
+  beforeEach(() => {
+    misconceptionObjectFactory = new MisconceptionObjectFactory();
+    misconceptionDict = {
+      id: '1',
+      name: 'test name',
+      notes: 'test notes',
+      feedback: 'test feedback',
+      must_be_addressed: true
+    };
+  });
+
+  it('should create a new misconception from dict', () => {
+    const misconception =
```

Comment:
I think the preferred way is to wrap inside `(..)` and break after `(`. Please fix it here and in the rest of the file. Thanks!

Reply:
Done.